app/Commands/Dev/Core/Repl.hs

```diff
@@ -26,7 +26,7 @@ parseText = Core.runParser replPath defaultModuleId
 
 runRepl :: forall r. (Members '[Embed IO, App] r) => CoreReplOptions -> Core.InfoTable -> Sem r ()
 runRepl opts tab = do
-  embed (putStr "> ")
+  putStr "> "
```

Comment:
I don't understand what happened here and why is it in this PR? We don't need `embed` anymore? This PR changes that all over the place - it should be separate PR doing this refactor.

Reply:
Done